packages/suins/sources/sales/pricing_config.move

```diff
@@ -16,17 +16,19 @@ const EPriceNotSet: u64 = 4;
 public struct Range(u64, u64) has copy, store, drop;
 
 /// A struct that holds the length range and the price of a service.
-public struct PricingConfig has copy, store, drop {
-    pricing: VecMap<Range, u64>,
-}
+public struct PricingConfig has store, drop { pricing: VecMap<Range, u64> }
```

Comment:
Would copy be useful in allowing user to check the current config? And add a pub read fun. If this won't be changed often and will be displayed in docs, then not needed